### PR TITLE
DS-143 fix: Revise ahb videos and remove outdated and broken items

### PR DESCRIPTION
**Pull request by froboy**
https://yusa.atlassian.net/browse/DS-143

changes will only occur on new install. @podarok should we have an upgrade path here?

### CODE REVIEW

File: modules/openy_demo_ahb/config/install/migrate_plus.migration.openy_demo_entity_ahb.yml [hunk 1]
```diff
@@ -16,251 +16,110 @@ source:
...
+      field_ahb_video: 'https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk&index=1&t=10s'
       field_ahb_pages: '/node/add/blog'
       field_ahb_visibility: 'include'
     -
-      id: 21
+      id: 10
       field_ahb_title: 'YMCA Website Services: Taxonomy'
       field_ahb_description: |
-        <p>Can be found under Content (after filtering for blog content). It could be placed at the root of <a href="/content">/content</a> to be able to find easier, and/or add individual links to each section. Given there are a number of tutorials, suggest linking to the existing playlist on YouTube and have one link - <a href="https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk">https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk</a></p>
+        <p>Group pieces of related content together for tagging, filtering, sorting and grouping. <a href="https://ds-docs.y.org/docs/user-documentation/taxonomy/" target="_blank">Learn more in our docs</a>.</p>
```
Comment (podarok) [inline modules/openy_demo_ahb/config/install/migrate_plus.migration.openy_demo_entity_ahb.yml]:
```suggestion
        <p>Group pieces of related content together for tagging, filtering, sorting, and grouping. <a href="https://ds-docs.y.org/docs/user-documentation/taxonomy/" target="_blank">Learn more in our docs</a>.</p>
```

Comment (podarok) [reply modules/openy_demo_ahb/config/install/migrate_plus.migration.openy_demo_entity_ahb.yml]: I know, Standford comma dilemma....

**Review by podarok — APPROVED** [review]
Very small @froboy 
Could be ignored if you are opposite

**Comment by podarok** [comment]
We don't need upgrade path here @froboy

File: modules/openy_demo_ahb/config/install/migrate_plus.migration.openy_demo_entity_ahb.yml (hunk 1 above)
Comment (froboy) [reply modules/openy_demo_ahb/config/install/migrate_plus.migration.openy_demo_entity_ahb.yml]: https://github.com/open-y-subprojects/openy_demo_content/assets/238201/a8c49601-2bc9-415c-be02-a0ec019971ca

I copy and pasted from https://ds-docs.y.org/docs/user-documentation/taxonomy/... now I have another fix to fix.

**Comment by froboy** [comment]
Thanks for the feedback. This should be good to go now.

**Pull request merged**